packages/nextjs/pages/index.tsx

```diff
@@ -154,6 +153,11 @@ const Home: NextPage = () => {
                 Accelerate your dapp development using our pre-built components for common web3 use cases. Tailwind and
                 daisyUI to style your dapp and give it a modern and appealing design.
               </p>
+              <p className="m-auto text-center lg:text-left lg:mx-0 max-w-[300px] lg:max-w-none lg:pr-6 underline">
```

Comment:
NITPICK: Maybe we could `underline-offset-2` so that underline is a bit below, we can even also directly add `.link` to daisy them and use it here which we did at [scaffold-eth-2/@cd06684](https://github.com/scaffold-eth/scaffold-eth-2/commit/cd06684219217984556dd184a207acbbe33ea61b)

 but even one current looks not too bad

Reply:
Thx for the tip! I've just implemented it like in SE-2 creating the `.link` style for `daisyUI` 👌